.github/workflows/test.yml

```diff
@@ -51,17 +51,6 @@ jobs:
       - name: Build Dist 🔧
         run: yarn run build
 
-      - name: Build Storybook 📚
-        run: yarn workspaces foreach run build:storybook
-
-      - name: Run Accessibility Tests 💛
-        run: yarn workspaces foreach run storybook:axeOnly
-
-      - name: Percy Snapshots 📸
-        run: yarn workspaces foreach run snapshot
-        env:
-          PERCY_TOKEN: ${{ secrets.PERCY_TOKEN }}
```

Comment:
reminder to remove this from the secrets page

Reply:
done, thanks for catching